### PR TITLE
element retrieval: improve intro wording

### PR DESCRIPTION
**Pull request by andreastt**

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -3039,16 +3040,18 @@
 <section>
 <h2>Element Retrieval</h2>
 
-<p>The <a>Find Element</a> and <a>Find Elements</a> <a>commands</a>
- allows look up of individual and a collection of <a>elements</a>, respectively.
+<p>The <a>Find Element</a> and <a>Find Elements</a> commands
+ allow look up of individual- and collections of <a>elements</a>.
```
Comment (burg) [inline webdriver-spec.html]: I find this construction less clear than 'individual elements and collections of elements, respectively'.

File: webdriver-spec.html [hunk 2]
```diff
@@ -3039,16 +3040,18 @@
...
+ Element retrieval searches are performed
```
Comment (burg) [inline webdriver-spec.html]: You have not tied the term 'Element Retrieval' to a definition/command. So this does not follow from the previous sentence.

File: webdriver-spec.html [hunk 3]
```diff
@@ -3039,16 +3040,18 @@
...
+ using pre-order traversal of the document’s nodes
```
Comment (burg) [inline webdriver-spec.html]: Is there a reason to specify the traversal order, as opposed to a sort order on the results? Specifying a traversal seems overly restrictive to how selector matching might be performed.

Comment (andreastt) [inline webdriver-spec.html]: This is just a description of how querySelectorAll works, but I’m happy to leave this undescribed if you think that’s a better idea?

File: webdriver-spec.html (hunk 2 above)
Comment (andreastt) [inline webdriver-spec.html]: It’s the name of the chapter, not a definition as such?

File: webdriver-spec.html (hunk 1 above)
Comment (andreastt) [inline webdriver-spec.html]: I agree.  Fixed in 926aa874ce2a195fab4cf4c2ea34c9cd595a2daf.

**Comment by AutomatedTester** [comment]
landed in fe1c2d5

**Pull request closed without merging**